### PR TITLE
fix: update beacon proxy file

### PR DESCRIPTION
**Pull request by amelnytskyi**
# What :computer: 
* `chalk.green` causes an error when running script - removed it

# Why :hand:
* Causing failures as `chalk` not necessarily installed

### CODE REVIEW

File: docs/build/tooling/hardhat/hardhat-zksync-upgradable.md [hunk 1]
```diff
@@ -606,22 +606,20 @@ async function main() {
 
   const beaconAddress = await beacon.getAddress();
 
-  const boxBeaconProxy = await hre.zkUpgrades.deployBeaconProxy(deployer.zkWallet, beaconAddress, contract, [42]);
+  const boxBeaconProxy = await hre.zkUpgrades.deployBeaconProxy(deployer.zkWallet, beaconAddress, contract, []);
```
Comment (MexicanAce) [inline docs/build/tooling/hardhat/hardhat-zksync-upgradable.md:609]: This should work, can you post your `Box` smart contract? The one on this page is the following:

```solidity
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.16;
import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";


contract Box is Initializable{
    uint256 private value;
    uint256 private secondValue;
    uint256 private thirdValue;

    function initialize(uint256 initValue) public initializer {
        value = initValue;
    }

    // Reads the last stored value
    function retrieve() public view returns (uint256) {
        return value;
    }

    // Stores a new value in the contract
    function store(uint256 newValue) public {
        value = newValue;
        emit ValueChanged(newValue);
    }
    // Emitted when the stored value changes
    event ValueChanged(uint256 newValue);

}
```

File: docs/build/tooling/hardhat/hardhat-zksync-upgradable.md [hunk 2]
```diff
@@ -606,22 +606,20 @@ async function main() {
...
-  console.info(chalk.green("Successfully upgraded beacon Box to BoxV2 on address: ", beaconAddress));
+  console.info("Successfully upgraded beacon Box to BoxV2 on address: ", beaconAddress);
 
   const attachTo = new zk.ContractFactory<any[], Contract>(boxV2Implementation.abi, boxV2Implementation.bytecode, deployer.zkWallet, deployer.deploymentType);
   const upgradedBox = attachTo.attach(await boxBeaconProxy.getAddress());
 
   upgradedBox.connect(zkWallet);
   // wait some time before the next call
   await new Promise((resolve) => setTimeout(resolve, 2000));
-  const value = await upgradedBox.retrieve();
```
Comment (MexicanAce) [inline docs/build/tooling/hardhat/hardhat-zksync-upgradable.md:623]: This should also work based on the BoxV2 from this example. Can you double check that your BoxV2 has the following:

```solidity
    // Reads the last stored value and returns it with a prefix
    function retrieve() public view returns (string memory) {
        return string(abi.encodePacked("V2: ", uint2str(value)));
    }
```

Comment (amelnytskyi) [reply docs/build/tooling/hardhat/hardhat-zksync-upgradable.md]: Rollbacked changes - all works as expected

File: docs/build/tooling/hardhat/hardhat-zksync-upgradable.md (hunk 1 above)
Comment (amelnytskyi) [reply docs/build/tooling/hardhat/hardhat-zksync-upgradable.md]: Rollbacked changes - all works as expected
Sorry for misinformation, only `chalk.green` needs to be removed then

**Review by MexicanAce — APPROVED** [review]
LGTM

**Pull request merged**